src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertsArtworks.tsx

```diff
@@ -0,0 +1,317 @@
+            <Join separator={<Spacer y={2} />}>
+              <Flex flexWrap="wrap" gap={1}>
+                <CriteriaPills editable={false} />
+              </Flex>
+
+              {count > 0 && (
+                <>
+                  <Text variant="sm-display" color="black60">
+                    {count}
+                    {count > 1 ? " works" : " work"} currently on Artsy match
+                    your criteria.
+                    <Text variant="sm-display" color="black60">
+                      See our top picks for you:
+                    </Text>
+                  </Text>
+                  <Spacer y={2} />
+                </>
+              )}
+
+              <ArtworkGrid
+                artworks={alert.artworksConnection}
+                columnCount={2}
+                emptyStateComponent={<ArtworksGridEmptyState />}
+              />
+
+              {count > 0 && (
+                <Button
+                  // @ts-ignore
+                  as={RouterLink}
+                  to={href}
+                >
+                  See All Matching Works
+                </Button>
+              )}
+              <Button variant="secondaryBlack" onClick={onEditAlertClick}>
+                Edit Alert
+              </Button>
+            </Join>
+          </Flex>
+        </Modal>
+      </Media>
+    </>
+  )
+}
+
+const ArtworksGridEmptyState = () => {
+  return (
+    <Message width="100%" title="No matches">
+      There aren’t any works available that meet the criteria at this time.
+    </Message>
+  )
+}
+
+interface SavedSearchAlertsArtworksQueryRendererProps {
+  editAlertEntity: EditAlertEntity
+  onCloseClick?: () => void
+  onEditAlertClick: () => void
+}
+
+export const SavedSearchAlertsArtworksFragmentContainer = createFragmentContainer(
+  AlertArtworks,
+  {
+    alert: graphql`
+      fragment SavedSearchAlertsArtworks_alert on Alert {
+        internalID
+        acquireable
+        additionalGeneIDs
+        artistIDs
+        artistSeriesIDs
+        atAuction
+        attributionClass
+        colors
+        dimensionRange
+        sizes
+        width
+        height
+        inquireableOnly
+        locationCities
+        majorPeriods
+        materialsTerms
+        offerable
+        partnerIDs
+        priceRange
+      }
+    `,
+  }
+)
+
+export const SavedSearchAlertsArtworksQueryRenderer: React.FC<SavedSearchAlertsArtworksQueryRendererProps> = ({
+  editAlertEntity,
+  onCloseClick,
+  onEditAlertClick,
+}) => {
+  return (
+    <SystemQueryRenderer<SavedSearchAlertsArtworksQuery>
+      query={graphql`
+        query SavedSearchAlertsArtworksQuery($id: String!) {
+          me {
+            alert(id: $id) {
+              artworksConnection(first: 10) {
+                counts {
+                  total
+                }
+                ...ArtworkGrid_artworks
+              }
+              href
+              ...SavedSearchAlertsArtworks_alert @relay(mask: false)
```

Comment:
Is `@relay(mask: false)` here on purpose? The documentation says it's [not recommended](https://relay.dev/docs/api-reference/graphql-and-directives/#relaymask-boolean) to use `mask: false`, so it might be good to avoid it if possible.

Reply:
Unfortunately yes, we need it. I tried to avoid using it, but when I `remove @relay(mask: false)` and adjust the query it breaks the AlertProvider
We use the same pattern in  SavedSearchAlertEditForm - tried in this file as well and couldn't make it